tgui-next/packages/tgui/interfaces/StationAlertConsole.js

```diff
@@ -0,0 +1,59 @@
+import { Fragment } from 'inferno';
+import { act } from '../byond';
+import { Section, Button } from '../components';
+
+export const StationAlertConsole = props => {
+  const { state } = props;
+  const { config, data } = state;
+  const { ref } = config;
+  const categories = data.alarms || [];
+  const fire = categories['Fire'] || [];
+  const atmos = categories['Atmosphere'] || [];
+  const power = categories['Power'] || [];
+  return (
+    <Fragment>
+      <Section title="Fire Alarms">
+        <ul>
+          {fire.length === 0 && (
+            <li className="color-good">
+              Systems Nominal
+            </li>
+          )}
+          {fire.map(alert => (
+            <li key={alert}>
```

Comment:
Shouldn't these have some color ?

Reply:
Nah, they dont. We can add something to make it more clear though
![image](https://user-images.githubusercontent.com/33292112/69012150-e02a1f00-0972-11ea-89e9-6ffd4ef07c30.png)

Reply:
Alright, just my imagination then.